libs/order/state/src/selectors/order-entities.selector.ts

```diff
@@ -227,6 +231,15 @@ const createOrderEntitySelectors = <T extends DaffOrder = DaffOrder>() => {
       return index > -1 ? totals[index] : null
     }
   );
+  const selectOrderHasDiscount = createSelector(
+    selectEntities,
+    (orders, props) => {
+      const totals = selectOrderTotals.projector(orders, { id: props.id });
+      const index = totals.findIndex(total => total.type === DaffOrderTotalTypeEnum.Discount);
+
+      return index > -1;
```

Comment:
Shouldn't you also check that the discount totals is greater than 0? Seems like you can just select `selectOrderDiscountTotal` and check the value.

Reply:
yea, my bad. Thanks for catching that. Changed